virtcontainers/container.go

```diff
@@ -669,6 +672,7 @@ func createContainer(sandbox *Sandbox, contConfig ContainerConfig) (c *Container
 	defer func() {
 		if err != nil {
 			c.rollbackFailingContainerCreation()
+			c.sandbox.storage.deleteContainerResources(c.sandboxID, c.id, nil)
```

Comment:
There's a call to `c.removeResources()` from `rollbackFailingContainerCreation()`, so I would assume `c.sandbox.storage.deleteContainerResources()` is already part of `c.removeResources()`, right?
If it's not, make sure to add it to the rollback function instead of adding it here.

Reply:
yes you're right, I moved it to `rollbackFailingContainerCreation()` now :+1: